github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRCommitEvent/help.html

```diff
@@ -1,3 +1,4 @@
 <div>
-    Trigger when hash changed in comparison to previous checked hash.
+    Triggers when a new commit is pushed, or force-pushed, to a pull request
+    that has been built previously.
```

Comment:
Maybe keep technical side that it based on `hash` and place in brackets `(new commit is pushed...`?